frontend/src/screens/SinglePlaylist/index.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { Navbar } from '../../components'
+import useCustomContext from '../../customHooks/Hook';
+import styles from './singleplaylist.module.css';
+import ReactPlayer from 'react-player';
+import { useParams, Link } from 'react-router-dom';
+import { DELETE_PLAYLIST, REMOVE_ITEM_FROM_PLAYLIST } from '../../constants/type';
+
+const SinglePlaylist = () => {
+    const { state: { playlist }, dispatch } = useCustomContext();
+    const { id } = useParams();
+
+
+    const deleteFromPlaylist = (playlistId, id) => {
+        dispatch({ type: REMOVE_ITEM_FROM_PLAYLIST, payload: { playlistId, id } })
+    }
+
+    const deletePlaylist = (id) => {
+        dispatch({ type: DELETE_PLAYLIST, payload: id })
+    }
+
+    const findVideosofPlaylist = playlist.filter((one) => one.id === id);
```

Comment:
Could this be destructured for ease of writing name Id and all?
I this the filter would be sending array with one element right?
I am not sure about the result
This is more of question with thought

Reply:
This filter matches the playlist id with id in params and accordingly finds all videos in a given playlist which is iterated accordingly